src/preset_cli/cli/superset/sync/native/command.py

```diff
@@ -301,6 +308,19 @@ def get_charts_uuids(config: AssetConfig) -> Iterator[str]:
             yield child["meta"]["uuid"]
 
 
+def get_filters_dataset_uuids(config: AssetConfig) -> Set[str]:
+    """
+    Extract native filters dataset UUID from a dashboard config.
+    """
+    uuids = set()
+    for child in config["metadata"]["native_filter_configuration"]:
```

Comment:
This might actually fail for dashboard without either `metadata` nor `native_filter_configuration` attributes. Better to add

`if config.get('metadata') and config['metadata'].get('native_filter_configuration'):`